### DISCUSSION TITLE
Fake Data: Add fake log rows

### DISCUSSION THREAD

**Discussion by latere-a-latere** [grafana/grafana#59218] [Data Sources]
For testing out Loki dashboards I'd very much like a way to mock logs. The current built-in fake datasource only seems to support random numbers and file listings, an additional generator for log rows would be very useful.

There are two types of logs I want to emulate:
- Application logs (standard log4net/serilog stuff)
- Server access logs (W3C, IIS, Apache, etc.)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
